libcxx/utils/generate_feature_test_macro_components.py

```diff
@@ -882,7 +882,7 @@ def add_version_header(tc):
         },
         {
             "name": "__cpp_lib_ranges",
-            "values": {"c++20": 202207},
+            "values": {"c++20": 202211},  # DR P2602R2 Poison Pills are Too Toxic
```

Comment:
Preexisting: libc++'s `__movable_box` is actually _`copyable-box`_ in C++20 mode, but this feature-test macro is unconditionally bumped. Moreover, libc++ hasn't implemented the standardized `std::range_adaptor_closure`, so perhaps this macro shouldn't be even bumped to `202202L` at this moment.

I've submitted [LWG3931](https://cplusplus.github.io/LWG/issue3931) which is related. (The currently proposed resolution is not good. Don't implement it now.)

CC-ing @yronglin.